posixlib/src/main/scala/scala/scalanative/posix/sys/socket.scala

```diff
@@ -183,6 +183,22 @@ object socket {
            length: CSize,
            flags: CInt): CSSize = extern
 
+  @name("scalanative_sendto")
+  def sendto(socket: CInt,
+             buffer: Ptr[Byte],
+             length: CSize,
+             flags: CInt,
+             dest_addr: Ptr[sockaddr],
```

Comment:
Although, these are not 100% consistent in this file or across the bindings as a whole, these should follow Scala's toggle case convention like `destAddr`.

Reply:
This PR also needs a rebase as all the macro constants are now all lower case. This was changed after this PR as they were not consistent.

Reply:
Eric,  thank you for the review.  For the sake of making progress,
I propose that this PR be accepted as is and I will do an immediate
PR to fix up your two concerns.  

In a different world,  I would do the edits here, but I 
do not have suitable privileges. By my understanding,
only the original author can do that. Given the months
long delay, that person may no longer be listening to Scala Native.


This avoids a long wait and disincentive to the original contributor 
because of Scala Native internal churn.

re: `dest_addr: Ptr[sockaddr],` IIUC dst_addr => destAddr and sockaddr remain stat.

Re-basing constants should be straightforward.

Reply:
@mkotsbak I don't know if you have seem this yet since I just did the review today. Are you able to rebase and do the updates?

Reply:
@ekrich   Since  this code does not touch any of the `@name` lines
you recently brought into consistency,  I believe that a normal merge
will retain your edits.  That is, since there are no conflicts reported by
GitHub, why is a full re-base necessary?  Am I missing something?

If I understand correctly, the requested change is to change
`dest_addr` to `destAddr`.  The longstanding convention, for better or worse,
in that file is to use snake_case, c.f. `address_len` on the next line.
Seems to me that the @mkotsbak did a good job of following prior art.

I do not want to be giving conflicting advice, and am the junior contributor,
so will defer to your judgement.

Reply:
@ekrich Good catch about the `@name`. They would make an immediate
merge more difficult.

I believe that a Git rebase will fix the UPPERCASE to lowercase `@name` issue without requiring manual edits.

That leaves the `destAddr`.  The long standing convention in that file is
to use snake_case: c.f. the many extant places which use `address_len`.
Seems to me that @mkotsbak did a good job of following prior art and
that the requested change to `destAddr` actually introduces inconsistency.

I personally doubt the value of a sweep through the file to eliminate all use of  snake_case but if that were going to be done, we probably agree that
it should be in a separate PR.

Reply:
Yes, that should match the C API name.

Reply:
@Mergifyio rebase

Reply:
Forgot about this. I will look at it soon.